File: mortage.py
__author__ = 'Parth'
import math

print("What is the principle/initial loan?")
loan = input()

print("What is the Interest Rate?")
intrest_rate = input()
monthly_interest = intrest_rate/1200

print("What is the fixed rate time?")
year = input()
month = year * 12


sec1 = loan * (monthly_interest * (math.pow (1 + monthly_interest, month)))
sec2 = (math.pow(1 + monthly_interest, month-1))
mortage = sec1/sec2
print "Monthly fee: ", mortage

File: mortage.ml
let () = print_string "Principle/initial loan: " in
let loan = read_double();;

let () = print_string "Interest Rate?" in
let intrest_year = read_double();;
let monthly_interest = intrest_year / 12;;

let () = print_string "Year" in
let time = read_double();;
let month = time*12

let monthly_payment = loan * (monthly_interest(1 + monthly_interest)^month)/(monthly_interest(1 + monthly_interest)^month - 1);;

print_string "Month payment";;
print_int monthly_payment;;